src/actions/slack/legacy_slack/slack.ts

```diff
@@ -28,9 +28,13 @@ https://github.com/looker/actions/blob/master/src/actions/slack/legacy_slack/REA
 
   async form(request: Hub.ActionRequest) {
     const form = new Hub.ActionForm()
+    let channelType = "channels"
```

Comment:
nit: this can be simplified as

```js
request.formParams.channelType !== undefined && request.formParams.channelType === "users" ? "users" : "channels"
```

wait: wouldn't it just be

```js
request.formParams.channelType || "channels"
```

Reply:
linter didn't like the second solution